src/Application/Components/Mint/MintShop.jsx

```diff
@@ -11,18 +11,15 @@ const MintShop = React.createClass({
     let message;
     let onUpgrade;
     if (nextDenomination) {
-      message =
-        `next denomination: ${nextDenomination.id}, ` +
-        `income per click: ${nextDenomination.incomePerClick}, ` +
-        `price to unlock: ${nextDenomination.priceToUnlock}`;
+      message = `Upgrade Denomination for $${nextDenomination.priceToUnlock}`;
```

Comment:
This confused RJ and I at first, do you have an opinion on using the same style of button as the Real Estate menu?
i.e. Image, name, price, and maybe benefit.